src/main/java/tay/tayApplication.java

```diff
@@ -51,8 +53,12 @@ public void run(final tayConfiguration configuration,
 
         EventResource eventResource = new EventResource();
         environment.jersey().register(eventResource);
-
-
+        
```

Comment:
can you merge master into your branch, and then move these to the appropriate sections?